jericho/jericho.py

```diff
@@ -321,27 +321,11 @@ def _load_bindings(md5hash):
     - `grammar`: List of action templates for the game.
     - `max_word_length`: Maximum number of characters per word recognized by the parser.
 
-    :param rom: Path or name of rom to load.
-    :type rom: string
+    :param md5hash: A game's md5 hash.
+    :type md5hash: string
 
     :returns: Dictionary containing game-specific bindings if it exists.
               Otherwise, an empty dictionary.
-
```

Comment:
Moved this example in the docstring of the new property `jericho.FrotzEnv.bindings`.